Sources/Implementation/DefaultDatafileHandler.swift

```diff
@@ -110,27 +110,38 @@ class DefaultDatafileHandler: OPTDatafileHandler {
             guard let request = self.getRequest(sdkKey: sdkKey) else { return }
             
             let task = session.downloadTask(with: request) { (url, response, error) in
-                var result = OptimizelyResult<Data?>.failure(.datafileDownloadFailed("Failed to parse"))
+                var result = OptimizelyResult<Data?>.failure(.datafileLoadingFailed(sdkKey))
+
+                let returnCached = { () in
+                    if let data = self.loadSavedDatafile(sdkKey: sdkKey) {
+                        result = .success(data)
+                    }
+                }
                 
                 if error != nil {
                     self.logger.e(error.debugDescription)
                     result = .failure(.datafileDownloadFailed(error.debugDescription))
```

Comment:
This line should be removed.

Reply:
why?  i want to add the error discription. if there is no cache file available, that is what is returned.

Reply:
You're right. I was confused :)